crowbar_framework/app/models/service_object.rb

```diff
@@ -1543,9 +1561,10 @@ def run_remote_chef_client(node, command, logfile_name)
 
   private
 
-  def wait_for_chef_clients(node)
-    unless RemoteNode.chef_ready?(node, 1200)
-      STDERR.puts "Waiting for already running chef-clients on #{node} failed"
+  def wait_for_chef_clients(node_name)
+    @logger.debug("wait_for_chef_clients: Waiting for already running chef-clients on #{node_name}.")
+    unless RemoteNode.chef_ready?(node_name, 1200)
```

Comment:
@vuntz Are you confident about the other patches related to the fix?

I fear the worms. As this is really not related to the fix of [857375](https://bugzilla.suse.com/show_bug.cgi?id=857375), can we just drop the logging improvement commit from here and reopen another PR for that? Would be good to get the other stuff merged...